### DISCUSSION TITLE
Disable Body and :root elements styles

### DISCUSSION THREAD

**Discussion by Canic** [mantinedev/mantine#6171] [Feature requests]
Hello,

We're currently integrating Mantine into our website, which runs on various web frameworks such as Bootstrap 3 or Tailwind. Mantine will serve as the top layer within a WordPress plugin.

I've been unable to locate a setting to deactivate the styles applied to the Body and :root elements, like font-family, for instance. Presently, Mantine overrides these styles on our website.

It would be greatly appreciated if an option could be provided to disable these overrides without having to specify a particular font-family.

**Comment by rtivital** [comment]
You can fully control styles that are imported in your application with https://mantine.dev/styles/css-files-list/
